layers/+readers/pdf/README.org

```diff
@@ -49,6 +49,24 @@ If you use a Debian based system such as Ubuntu, you can skip this step, as
 =pdf-tools= installation asks whether you want it to try to download
 automatically all dependencies using =apt-get=.
 
+On macOS, the default instructions might not work. There is a brew tap at dunn/emacs that contains pdf-tools, but it did not work for me. I used the following successfully. Note that your PATH should be set correctly in Emacs, and that pdf-tools does not work in terminal (GUI-only).
+
+#+BEGIN_SRC
+brew install poppler automake
+#+END_SRC
+
+Then in emacs run the following:
+
+#+BEGIN_SRC
+(use-package pdf-tools
```

Comment:
Can you explain why you require a `use-package` declaration? Is it just for setting the variable? This could have been set much more easier during the layer declaration i.e. `(pdf variables: pdf-info-epdfinfo-program)`.

Reply:
Truth to tell, I copied this off stackoverflow, and don’t know if it’s necessary. (But it certainly worked for me.)  I figured people can improve on it if they wish. I personally found pdf-tools to be glacial, and did not investigate it further. I’ll incorporate your suggestion as an alternative.